### DISCUSSION TITLE
단검 여러개 발사 시 상황

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#160] [Q&A]
단검 여러개 발사할 때 어떤 식으로 발사할까요?
1. 동시에 여러개 단검이 겹치며 발사
2. 단검간의 시간 간격 두고 발사
3. 바라보는 벡터 +-작은 값으로 하여 단검이 겹치지 않게 발사
고려해야할 점 투사체가 최대일 때

**Comment by rapa1130** [comment]
뱀파이어 서바이벌에선 시간간격을 두고 발사되긴 합니다. 하지만 이렇게 디테일한거까지 다른 이유가 있지 않는 한 꼭 따라해야 하는 건 아니라고 생각합니다. 1번 케이스로 한다고 하더라도 저희가 추후 개발할 다른 무기와 크게 겹치지는 않기 때문에 그렇게 해도 괜찮습니다. 따라서 편하신대로 임의로 발사 방식을 선택하셔도 괜찮습니다.
 
+ 3번은 동시에 발사하되 위치에 차이를 둔다는 의미인건가요?

**Reply by seong0929** [reply]
- 3번은 동시에 발사하되 위치에 차이를 둔다는 의미인건가요? 네 맞아요
저도 굳이 꼭 따라할 필요가 없지만, 설명이 없어 어떻게 해야할 지 몰라서 물어보거예요.
